<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(180, 180, 180, 255))</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>781</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../images/logo_720.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color:black;

font: italic 28pt &quot;Harlow Solid&quot;;</string>
       </property>
       <property name="text">
        <string>Admin Preferences Menu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>401</width>
      <height>195</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>50</number>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_main">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>122</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);
}
QPushButton:hover {
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255)
}</string>
       </property>
       <property name="text">
        <string>Main Menu</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button_exit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>122</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);

}
QPushButton:hover {
color:black;
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

}</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button_app">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>122</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);
}
QPushButton:hover {
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255)
}</string>
       </property>
       <property name="text">
        <string>Applications</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_mentor">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>122</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);
}
QPushButton:hover {
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255)
}</string>
       </property>
       <property name="text">
        <string>Mentor Meeting</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_int">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>122</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);
}
QPushButton:hover {
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255)
}
</string>
       </property>
       <property name="text">
        <string>Interviews</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="button_admin">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>122</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid rgb(171, 171, 171);
}
QPushButton:hover {
    background-color:rgb(255, 226, 229);
}
QPushButton:pressed {
    background-color: rgb(218, 30, 60);
	color: rgb(255, 255, 255)
}</string>
       </property>
       <property name="text">
        <string>Admin Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
